pyloras/_loras.py

```diff
@@ -128,8 +133,7 @@ def _initialize_params(self, X, y, rng):
         self.n_affine_ = f_size if self.n_affine is None else self.n_affine
 
         if self.manifold_learner:
-            self._check_2d_manifold_learner()
-            self.manifold_learner_ = self.manifold_learner
+            self.manifold_learner_ = self._check_2d_manifold_learner()
```

Comment:
You forgot to add the `clone` function inside of `_check_2d_manifold_learner` and return it as the method's return value.

Reply:
should it be 
```
def _check_2d_manifold_learner(self):
        if (not hasattr(self.manifold_learner, "fit_transform") or
                not hasattr(self.manifold_learner, "set_params")):
            raise ValueError(
                "The 2d manifold learner must implement the ``fit_transform`` "
                "and ``set_params`` methods"
            )
        return clone(self.manifold_learner)
```
??

Reply:
Yes, that is correct.